modules/ossm-add-project-member-roll-resource-console.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/installing-ossm.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-add-project-member-roll-recourse-console_{context}"]
+= Adding or removing projects from the service mesh using ServiceMeshMemberRoll resource with the web console
+
+* You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll` resource.
+
+* The `ServiceMeshMemberRoll` resource is deleted when its corresponding `ServiceMeshControlPlane` resource is deleted.
+
+.Prerequisites
+* An installed, verified {SMProductName} Operator.
+* An existing `ServiceMeshMemberRoll` resource.
+* Name of the project with the `ServiceMeshMemberRoll` resource.
+* Names of the projects you want to add or remove from the mesh.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Click the *Project* menu and choose the project where your `ServiceMeshControlPlane` resource is deployed from the list, for example `istio-system`.
+
+. Click the {SMProductName} Operator.
+
+. Click the *Istio Service Mesh Member Roll* tab.
+
+. Click the `default` link.
+
+. Click the YAML tab.
+
+. Modify the YAML to add or remove projects as members.  You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll` resource.
```

Comment:
What if the SMMR has no namespaces at this point? In that case, the SMMR most likely won't contain either the `spec` field or the `members` sub-field.

I think you should therefore specify how exactly the user has to "modify the YAML to add or remove projects".

You need to show the same snippet that's shown in the next section ("Adding or removing projects from the service mesh using ServiceMeshMemberRoll resource with the CLI"):
```
apiVersion: maistra.io/v1
kind: ServiceMeshMemberRoll
metadata:
  name: default
  namespace: istio-system #control plane project
spec:
  members:
    # a list of projects joined into the service mesh
    - your-project-name
    - another-project-name
```

Or, since the Web Console sections comes before the CLI section, maybe just move the snippet here and remove it from the CLI section.

Reply:
done